### PR TITLE
fix(config-ui): fix `createdAt` and `lastPipeline` fields

### LINKED ISSUE #4654: [Feature][Config UI] Adding more info to the project list
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-devlake/issues?q=is%3Aissue) and found no similar feature requirement.

### Use case

Screen Shot 2023-03-14 at 2 37 33 PM

Right now the Project list doesn't have any additional information, such as if this project has a Blueprint, and we would like to add more info on each row to help users identify the status of a particular project. 

### Description
New:
Screen Shot 2023-07-11 at 4 31 54 PM

### Related issues

_No response_

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by d4x1**
### ⚠️ Pre Checklist

> Please complete _ALL_ items in this checklist, and remove before submitting

- [x] I have read through the [Contributing Documentation](https://devlake.apache.org/community/).
- [x] I have added relevant tests.
- [x] I have added relevant documentation.
- [x] I will add labels to the PR, such as `pr-type/bug-fix`, `pr-type/feature-development`, etc.

### Summar
1. add `createdAt` field
2. rename `latestPipeline` to `lastPipeline`
3. fix potential error when quering projects with `models.ApiOutProject`

### Does this close any open issues?
Closes #4654 

### Screenshots
image

### Other Information
None.

cc @mintsweet @klesh @abeizn

### CODE REVIEW

File: backend/core/models/project.go [hunk 1]
```diff
@@ -62,8 +62,8 @@ type ApiInputProject struct {
 }
 
 type ApiOutputProject struct {
-	BaseProject    `mapstructure:",squash"`
-	Metrics        *[]BaseMetric `json:"metrics" mapstructure:"metrics"`
-	Blueprint      *Blueprint    `json:"blueprint" mapstructure:"blueprint"`
-	LatestPipeLine *Pipeline     `json:"latest_pipeline,omitempty" mapstructure:"latest_pipeline"`
+	Project      `mapstructure:",squash"`
+	Metrics      *[]BaseMetric `json:"metrics" mapstructure:"metrics"`
```
Comment (klesh) [inline backend/core/models/project.go:66]: pointer to an array, it doesn't seem right, did it meant to be `[]*BaseMetric`?

File: backend/server/services/project.go [hunk 2]
```diff
@@ -350,9 +351,9 @@ func refreshProjectMetrics(tx dal.Transaction, projectInput *models.ApiInputProj
 	return nil
 }
 
-func makeProjectOutput(baseProject *models.BaseProject, withLatestPipeLine bool) (*models.ApiOutputProject, errors.Error) {
+func makeProjectOutput(project *models.Project, withLastPipeline bool) (*models.ApiOutputProject, errors.Error) {
 	projectOutput := &models.ApiOutputProject{}
-	projectOutput.BaseProject = *baseProject
+	projectOutput.Project = *project
```
Comment (klesh) [inline backend/server/services/project.go:356]: Maybe `projectOutput.Project` can be a pointer

Comment (d4x1) [reply backend/server/services/project.go:356]: There are two reasons:
1. `Project` is the core field of `ApiOutputProject`, and it shouldn't be nil.
2. Keep the original form [BaseProject](https://github.com/apache/incubator-devlake/blob/main/backend/core/models/project.go#L65).

File: backend/core/models/project.go (hunk 1 above)
Comment (d4x1) [reply backend/core/models/project.go:66]: @klesh 
I think it's not a correct usage(but it is not introduced in the current PR ). I'will fix it.

Comment (d4x1) [reply backend/core/models/project.go:66]: image

**Review by klesh — APPROVED** [review]

**Pull request merged**